pkg/collector/corechecks/cluster/kubernetesapiserver/kubernetes_eventbundle.go

```diff
@@ -64,6 +66,9 @@ func (b *kubernetesEventBundle) formatEvents(taggerInstance tagger.Component) (e
 	readableKey := buildReadableKey(b.involvedObject)
 	tags := getInvolvedObjectTags(b.involvedObject, taggerInstance)
 	tags = append(tags, fmt.Sprintf("source_component:%s", b.component))
+	tags = append(tags, "true_source:kubernetes")
```

Comment:
I think `true_source` name could be improved, perhaps not necessary. We already have Kubernetes tags attached to these events, but in case we want it, something like `orchestrator:kubernetes` would make sense.